keyboards/snagpad/keymaps/default/keymap.c

```diff
@@ -0,0 +1,63 @@
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+	LAYOUT_ortho_5x4(
+		KC_NLCK, KC_PSLS, KC_PAST, KC_PMNS, 
+		KC_P7, KC_P8, KC_P9, KC_PPLS, 
+		KC_P4, KC_P5, KC_P6, KC_PPLS, 
+		KC_P1, KC_P2, KC_P3, KC_PENT, 
+		KC_P0, KC_P0, KC_PDOT, KC_PENT),
+
+  LAYOUT_numpad_5x4(
```

Comment:
You'd want to change this too, or it will be offset by one.

```c
  LAYOUT_numpad_5x4(
    KC_NLCK, KC_PSLS, KC_PAST, KC_PMNS, 
    KC_P7, KC_P8, KC_P9, 
    KC_P4, KC_P5, KC_P6, KC_PPLS, 
    KC_P1, KC_P2, KC_P3, 
    KC_P0,        KC_PDOT, KC_PENT),
```